client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { PLAN_JETPACK_COMPLETE } from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import Button from 'calypso/components/forms/form-button';
+import getJetpackAdminUrl from 'calypso/state/sites/selectors/get-jetpack-admin-url';
+import { getSelectedSiteSlug, getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { buildCheckoutURL } from '../../../get-purchase-url-callback';
+
+import './style.scss';
+
+const CtaButtons = () => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const siteId = useSelector( getSelectedSiteId );
+
+	const stayFreeURL = useSelector(
+		( state ) => getJetpackAdminUrl( state, siteId ) ?? 'https://jetpack.com/'
+	);
+
+	return (
+		<div className="jetpack-complete-page__cta-buttons">
+			<Button
+				className="jetpack-complete-page__get-complete-button"
+				primary
+				href={
+					siteSlug
```

Comment:
Maybe we could still use a `const` here, to have it a bit cleaner. You can define it above like:
`const checkoutUrl = siteSlug ? buildCheckoutURL...`

Reply:
+1 to this 🙂